Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -290,7 +290,13 @@
         {
           "Id": "SEC101/036",
           "Name": "DoNotExposePlaintextSecrets/MySqlConnectionString",
-          "ContentsRegex": "$SEC101/036.MySqlConnectionStringAdo",
+          "SingleLineRegexes": [
```

Comment:
>SingleLineRegexes [](http://example.com/codeflow?start=11&length=17)

One point about this model, the *first* match is critical. if it doesn't match, nothing else happens. :) and so the first match must be something that is *required* and ideally is something that is not frequent, i.e., is sufficiently distinct to help keep candidate numbers down.